packages/hash/frontend/src/theme/components/inputs/MuiOutlinedInputThemeOptions.ts

```diff
@@ -0,0 +1,48 @@
+import { Components, Theme } from "@mui/material";
+
+export const MuiOutlinedInputThemeOptions: Components<Theme>["MuiOutlinedInput"] =
+  {
+    defaultProps: {
+      notched: false,
+    },
+    styleOverrides: {
+      root: ({ theme }) => ({
+        color: "inherit",
+        lineHeight: "18px",
+        borderRadius: "6px",
+        "&:hover": {
+          "& .MuiOutlinedInput-notchedOutline": {
+            borderColor: theme.palette.gray[30],
+          },
+        },
+        "&.MuiOutlinedInput-root": {
+          paddingRight: "unset",
+          paddingLeft: theme.spacing(1.5),
+        },
+        "&.Mui-focused": {
+          "& .MuiOutlinedInput-notchedOutline": {
```

Comment:
This is fine for now but in future it may be worth considering accessing such classnames in a more type-safe way:
```suggestion
          [`& .${outlinedInputClasses.notchedOutline}`]: {
```

There's a lot of opportunity for hard-to-spot typos in classnames such as these.